pkg/controller/istio/controller.go

```diff
@@ -307,7 +307,7 @@ func (c *Controller) createIstioConfigurationForServiceEntry(oneagent dynatracev
 		return err
 	}
 	if sve == nil {
-		return fmt.Errorf("could not create service entry with spec %v", serviceEntry.Spec)
+		return fmt.Errorf("Could not create service entry with spec %v", serviceEntry.Spec)
```

Comment:
golang error messages should start with lower case
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Changed to lower case